recipes/sess_i/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sess_i" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/sess_i-{{ version }}.tar.gz
+  sha256: 3c59317b9e02ebf6c1bd8602bc83a70a0ac3835e4c0f01d19a9de1d452e9cbfd
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+  - {{ pin_subpackage('sess_i', max_pin="x") }}
+
+requirements:
+  host:
+    - python >=3.8,<4.0
```

Comment:
Please have a look at our documentation on `noarch: python` packages, specifically the use of the `python_min` jinja variable in the `host`, `run` and `test` sections:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python